### DISCUSSION TITLE
如何代理“寄生管理器”？

### DISCUSSION THREAD

**Discussion by GongT** [LSPosed/LSPosed#2390] [Q&A]
我用v2ray（别的应该差不多

不知道是什么原理实现的，看起来不是一个app，所以分应用代理列表里没有。

有没有什么办法做到（

**Comment by Howard20181** [comment]
代理Shell（包名为com.android.shell）就可以了
但是大多数代理软件过滤列表中没有这个应用，没有办法，你只能不分应用代理或者使用选择不经过代理的应用而不是选择经过代理的应用

**Comment by GongT** [comment]
看来只能更新的时候暂时取消分应用了
